Dockerfile

```diff
@@ -1,11 +1,33 @@
-FROM python:2.7 
+# getting the base Image
+FROM python:3.6
+
 MAINTAINER Ana Balica <ana.balica@gmail.com>
+
+# Updated by
+LABEL HUSSAIN="husainattar110@gmail.com"
```

Comment:
I am not sure if this required, lots of people may commit on the Dockerfile in the future, we do not want to create unnecessary noise here.

Reply:
I actually used because in future some needs help in changing or understand the docker file.
I tried to contact ana I guess during my docker issues but didn't get so I thought to include my mail for any help or maintenance something. Btw I can change it right now.